### DISCUSSION TITLE
有Layui的QQ或微信交流群吗？

### DISCUSSION THREAD

**Discussion by gh-wcw** [layui/layui#1842] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

v2.9.8

### 问题描述

有Layui的QQ或微信交流群吗？

### 业务代码

```auto
-
```

### 浏览器

-

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by Sight-wcg** [comment]
没有哦，Github [Discussions](https://github.com/layui/layui/discussions) 就是讨论 layui 相关问题最好的地方。
提问之前，请先阅读 https://github.com/layui/layui/blob/main/CONTRIBUTING.md
